File: Algorithms/Medium/P2414-Length_of_the_Longest_Alphabetical_Continuous_Substring/p2414.ac.c
int longestContinuousSubstring(char * s){
    const int lens = strlen(s);
    int ans = lens == 0 ? 0 : 1;
    int tmp_ans = ans;
    for (int i = 1; i < lens + 1; ++i) {
        if (s[i] == s[i - 1] + 1) {
            ++tmp_ans;
            ans = tmp_ans > ans ? tmp_ans : ans;
        } else {
            tmp_ans = 1;
        }
    }
    return ans;
}

File: Algorithms/Medium/P2414-Length_of_the_Longest_Alphabetical_Continuous_Substring/p2414.ac.ts
function longestContinuousSubstring(s: string): number {
    let ans: number = s.length == 0 ? 0 : 1;
    let tmpAns: number = ans;
    const charCode = (s: string) => s.charCodeAt(0) - 97;
    for (let i = 1; i < s.length; ++i) {
        if (charCode(s[i]) == charCode(s[i - 1]) + 1) {
            ++tmpAns;
            ans = tmpAns > ans ? tmpAns : ans;
        } else {
            tmpAns = 1;
        }
    }
    return ans;
};
